redisinsight/ui/src/electron/components/ConfigElectron/ConfigElectron.tsx

```diff
@@ -27,9 +28,9 @@ const ConfigElectron = () => {
         dispatch(fetchAccountInfo())
       }
 
-      if (status === CloudAuthStatus.Failed) {
+      if (status === CloudAuthStatus.Failed && error) {
```

Comment:
What if no error? Do you have default one or it request will fail silently

Reply:
added default one